tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRules.cs

```diff
@@ -245,9 +245,9 @@ public static Result IsMongoDb(this MockSpan span) => Result.FromSpan(span)
                 .Matches(Name, "mongodb.query")
                 .Matches(Type, "mongodb"))
             .Tags(s => s
-                .IsOptional("db.name")
-                .IsOptional("mongodb.collection")
-                .IsOptional("mongodb.query")
+                .IsOptional("db.instance")
+                .IsOptional("db.mongodb.collection")
+                .IsOptional("db.statement")
```

Comment:
[OTEL](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#mongodb) has `db.system`=`"mongodb"`. I think we should add that as well